english/movie-ripoffs/zoolander/zoolander.md

```diff
@@ -0,0 +1,24 @@
+As you approach the shaggy haired hippy, it is immediately obvious that the fellow is "high as a kite",
+his eyes unsuccessfully attempting to focus on you as he inhales deeply from a half smoked joint.
+"Who are you, and what the hell is this place?" you ask loudly, hoping to penetrate his drug-fogged stupor.
+
+"Chill Bro - you're ruining the vibe!" whines the stoner.  "I'm Hansel by the way." he adds, waving vaguely.
+
+"Where am I, and how do I get out of here?" you ask the red-eyed derelict, with a sense of increasing futility.
+"Dude, that's awesome!" Hansel responds wide-eyed.  "You're lost! I'm lost! We're like cosmic twins or something!".
+
+"But how did you get here?" you demand irritably, attempting to keep the conversation on track.
+
+Looking puzzled Hansel continues. "Dunno Bro. I was climbing Mt Vesuvius and I slipped and fell.  Next thing I was here."
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.